firmware/boron-ins-fsm/inc/spark_wiring_string.cpp

```diff
@@ -116,39 +116,39 @@ String::String(unsigned char value, unsigned char base)
 {
 	init();
 	char buf[9];
-	utoa(value, buf, base);
+	string_convert::utoa(value, buf, base);
```

Comment:
Oh, do namespaces usually use underscores or camelcase?

Reply:
Discussed in call and agreed naming scheme should be fine